### ISSUE TITLE
I'd like to contribute design/branding work

### ISSUE THREAD

**Issue by juliankrispel** [lhorie/mithril.js#433]
Hi. I'm Julian.

I sometimes do design/branding in the open for [projects](https://github.com/chocolatey/chocolatey/issues/640) I  [love](https://github.com/swannodette/om/issues/215).

Mithril is awesome and I'd like to contribute. May I? :)

**Comment by kay-is** [comment]
Sounds like a good idea to me, the Mithril page could use a redesign. 
Also a usability clean-up, because sometimes links vanish while navigating the site.

**Comment by lhorie** [comment]
Hi @juliankrispel 

Thanks for reaching out! I took a look at your concepts for Om and Chocolatey, and I thought they were pretty nice.

I suppose you want to design a new logo to replace the borromean rings, and a redesign of the layout?

I quite like the rings as a metaphor to MVC, but I'd love to see some new ideas.

For the layout, if you want to hack at files, right now there's three files in the docs/layout folder (index.html, api.html and guide.html). api and guide are basically the same thing, just different menu. index is the homepage. The site is autogenerated via a grunt task from those files and the content is populated from the markdown files in the docs folder.

**Comment by juliankrispel** [comment]
Thanks @lhorie. Glad your on board.

Disclaimer: I totally intend to work with you on this. The outcome of this depends a great deal on the conversation we have, me you and anybody else involved.

Yes I'd like to redesign the logo in some form or another. That could just be a revamp of the rings, or something entirely else, I guess I need to have a think first, then add some ideas in here and discuss it.

I'll try to get the branding nailed down first and then mess around with the website.

Also, for reference. Mithril is from [LOTR](http://lotr.wikia.com/wiki/Mithril) right? Super awesome name!

**Comment by lhorie** [comment]
yep: http://en.wikipedia.org/wiki/Mithril 

The logo idea was actually inspired by the chainmail texture of the hobbit's mithril shirt in the movie :)

Looking forward to seeing your concepts

**Comment by barneycarroll** [comment]
@juliankrispel I think you could do wonders for this Julian. Mithril's API is incredibly light and elegant yet robust – just like the fictional metal – so it'd be nice if it's public presence reflected that in look and feel. 

The dwarves mined it but the elves made the best stuff out of it IIRC. I use the past tense – the Mithril mines are long closed. Any Mithril artifacts you come about are extremely valuable as a result. The metal, not the library.

**Comment by juliankrispel** [comment]
Not sure about wonders but I'll try my best ;). Can't wait to get started.

**Comment by tomekwi** [comment]
:+1: great idea Julian! Can't wait for your redesign :)

**Comment by DeepAnchor** [comment]
this is a great idea, props to you @juliankrispel. Your work on Om looks slick! Mithril is definitely a project that could benefit from a little extra marketing facelift IMHO, for a framework of its quality. Looking forward to this!

**Comment by JoeDRL** [comment]
Great idea!
Mirthil needs more lovin' right now and a redesign by a quality artist ike @juliankrispel is perfect.

**Comment by bewildergeist** [comment]
Sorry to see this died out, I totally support the intent. I love how fast and light-weight the site is, which perfectly mirrors Mithril itself, but the design betrays how new and modern the framework is.

I'm a frontend developer and would love to contribute style updates and improved responsive support for the site. Collaborating with @juliankrispel or another designer would be great, as I lean more towards implementation than design :)

Was this closed for a lack of activity or a change of mind, @lhorie?

**Comment by acconrad** [comment]
I actually do design (and am a front end dev).

I've done branding for [my current company](http://www.coachup.com) and I did the design for [ShareLatex](https://www.sharelatex.com/) and [my side project that I am the sole developer and designer of](https://peergym.herokuapp.com/). Let me know if interested as well

**Comment by lhorie** [comment]
@bewildergeist I closed it due to lack of activity. Design contributions would be welcome

**Comment by bewildergeist** [comment]
@lhorie Excellent. I think a key point is to keep the pages as light-weight as they are now, if not even lighter, which implicitly communicates Mithril's speed and simplicity. A site with a 4KB `style.css` is a precious rarity these days! :)

What is the actual workflow for rendering the site — it seems to be generated by this repository's [Gruntfile.js](https://github.com/lhorie/mithril.js/blob/next/Gruntfile.js) but the generated files are committed to the `gh-pages` branch on [lhorie/mithril](https://github.com/lhorie/mithril), right? So what would be the best way to fork and work on the site?

@acconrad Those are solid examples of browseable feature overviews, social proof and good iconography. @lhorie can be the judge of how much 'bling' to add to the site, but I can definitely see adapting some of those concepts.

**Comment by lhorie** [comment]
@bewildergeist yes, that's how the build works. Basically, the layout stuff is in this folder `https://github.com/lhorie/mithril.js/tree/next/docs/layout`. The grunt script basically generates HTML from the markdown files one folder up from that and then wraps them with the appropriate section layout file (either `guide.html` or `api.html`). There's some markup duplication between those two files and also `index.html`.

Easiest way to work on the layout would be to fork, modify those layout files and copy-paste changes to the other layout files, then send a PR.

**Comment by graforlock** [comment]
I wouldn't touch logo at at all, it's great; they symbolise both MVC as separate yet integral concerns and at the same time chainmail rings made of mithril. Makes perfect sense.

**Comment by juliankrispel** [comment]
I'd totally touch the logo :) But am really struggling to find the time.
Also bear in mind you can make a v2 of the logo.

**Comment by bewildergeist** [comment]
Great to see you back, @juliankrispel! I have a todo-list of under-the-hood website optimizations brewing in my head that I'll open a new issue for soon, which are all things that can be done before we start shifting any pixels around.

But I'd be very interested to see design-iteration ideas. I like the LOTR-references (which the community has also [picked][dave] [up][mor] [on][ark]) and the symbolism of the borromean rings/chainmail, so I vote for trying to iterate on that. Seems like there's potentiel for a neat liltte SVG-animated logo or somesuch?

[dave]: http://ratfactor.com/daves-guide-to-mithril-js
[mor]: https://github.com/gitter-badger/moria
[ark]: https://github.com/barneycarroll/arkenstone

**Comment by juliankrispel** [comment]
Ok I'll try to find some time :)
